src/extension.ts

```diff
@@ -144,9 +142,8 @@ export function activate(context: vscode.ExtensionContext) {
 				await vscode.commands.executeCommand("cosmosDB.refresh");
 			}
 		});
-	registerCommand('cosmosDB.api.getConnectionString', getConnectionString);
-	registerCommand('cosmosDB.api.getDatabase', getDatabase);
-	registerCommand('cosmosDB.api.revealTreeItem', revealTreeItem);
+
+	return createApiProvider([]);
```

Comment:
Looks like a new function was added. Is there a version bump required for azureextensionui?

Reply:
You mean this?:
https://github.com/Microsoft/vscode-cosmosdb/pull/930/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R911